src/style.css

```
@@ -0,0 +1,91 @@
+* {
+  box-sizing: border-box;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir.ttc);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir Heavy";
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 900;
+}
+
+body {
+  font-family: Avenir, Arial, Helvetica, sans-serif;
+  font-weight: 300;
+  margin-top: 20px;
+}
+
+.search-bar {
+  width: 100%;
+  position: relative;
+}
+
+.search-bar_input__big {
+  height: 70px;
+  width: 100%;
+  padding-left: 62px;
+  font-size: 16px;
+}
+
+.image-big {
+  position: absolute;
+  width: 19px;
+  height: 19px;
+  top: 25px;
+  left: 26px;
+}
+
+.search-bar_input__small {
+  position: relative;
+  width: 100%;
+  height: 42px;
+  padding-left: 33px;
+  font-size: 14px;
+}
+
+.image-small {
+  position: absolute;
+  z-index: 1;
+  width: 11px;
+  height: 11px;
+  left: 13px;
+  top: 15px;
+}
+
+.search-bar_input {
+  margin-bottom: 20px;
+  outline: none;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  font-family: Avenir, Arial, Helvetica, sans-serif;
```

```suggestion
  font-family: inherit;
```
You can try to inherit it from body

---

https://Kostiadon.github.io/layout_search-bar-airbnb
https://Kostiadon.github.io/layout_search-bar-airbnb/report/html_report